### PR TITLE
adjustments for buttons on record revealed pages

### PR DESCRIPTION
**Pull request by JohnHeeryTNA**
## About these changes

Adjusted display types for record revealed buttons.

## How to check these changes

Please check the Record revealed pages are displaying correctly across viewports

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [ ] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: sass/includes/_record-matters.scss [hunk 1]
```diff
@@ -63,13 +63,14 @@
     &__buttons {
         margin-top: 1rem;
         text-align: center;
-        display: grid;
+        display: flex;
         grid-template-columns: 1fr 1fr 1fr;
         grid-gap: 1.5rem;
```
Comment (ahosgood) [inline sass/includes/_record-matters.scss:68]: If we change to `display: flex;` then we don't need `grid-template-columns` and `grid-gap`

File: sass/includes/_record-matters.scss [hunk 2]
```diff
@@ -63,13 +63,14 @@
...
 
         @media (max-width: $screen__md) {
             grid-template-columns: 1fr;
             grid-gap: 0;
```
Comment (ahosgood) [inline sass/includes/_record-matters.scss:72]: Again, we can remove these rules now

Comment (JohnHeeryTNA) [reply sass/includes/_record-matters.scss]: I think we need to keep `grid-gap: 1.5rem;` in, as the buttons will come together without it. Unless there's a way that we can add standardised margins to buttons?

File: sass/includes/_record-matters.scss (hunk 1 above)
Comment (JohnHeeryTNA) [reply sass/includes/_record-matters.scss]: I think we need to keep `grid-gap: 1.5rem;` in, as the buttons will come together without it. Unless there's a way that we can add standardised margins to buttons?

Comment (ahosgood) [reply sass/includes/_record-matters.scss]: Does `grid-gap` do anything without `display: grid;`?

Comment (JohnHeeryTNA) [reply sass/includes/_record-matters.scss]: `grid-gap` will still apply with `display:flex`
https://css-tricks.com/minding-the-gap/

Comment (ahosgood) [reply sass/includes/_record-matters.scss]: Wow - I didn't know that. Thanks for the link 🙌🏼

**Pull request merged**